CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
+## 1.0.0 (Unreleased)
+
+We are bumping the version of the boundary terraform provider to 1.0.0 and will release new versions of the provider at its own cadence instead of keeping it in lockstep with Boundary.
+
+### Bug Fixes
+
+* During `terraform apply`, do not update existing user account passwords when the password field is updated in tf file.
+  ([Issue](https://github.com/hashicorp/terraform-provider-boundary/issues/71))
+  ([PR](https://github.com/hashicorp/terraform-provider-boundary/pull/70))
+
 ## 0.1.4 (January 14, 2021)
 
-Boundary 0.1.4 release ([see boundary changelog](https://github.com/hashicorp/boundary/blob/main/CHANGELOG.md#014-20210105))
+### New and Improved
+
+Update to `Boundary API 0.0.3` and `Boundary 0.1.4` ([see boundary changelog](https://github.com/hashicorp/boundary/blob/main/CHANGELOG.md#014-20210105))
```

Comment:
I'm not sure the best way to say this but it strikes me as a bit confusing because it suggests that the Boundary API itself is versioned, which it kind of is, but not 0.0.3. Also I don't think it needs the code quotes. Maybe: "Update to the Boundary Go SDK v0.0.3". We probably don't need to say it's against Boundary 0.1.4 because the TF provider implements what the Go SDK exposes.